test/js/buffer/triangle_element_buffer.test.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
 
 var test = require('prova');
-var Buffer = require('../../../js/data/buffer/buffer');
 var TriangleElementBuffer = require('../../../js/data/buffer/triangle_element_buffer');
 
 test('TriangleElementBuffer', function(t) {
-    var buf = new Buffer();
+    var buf = new TriangleElementBuffer();
     t.ok(new TriangleElementBuffer(buf), 'default buffer');
```

Comment:
This is a strange test but rewriting it feels out of scope for this PR. `Buffer2` has [good test coverage](https://github.com/mapbox/mapbox-gl-js/pull/1526/files#diff-4e94531e9a19c46e6d705f5a53c60dabR1).